src/components/utils/localesMapping.js

```diff
@@ -0,0 +1,145 @@
+export const CONFIRM_PREVIEWS_LOCALES_MAPPING = {
```

Comment:
Looks good to me. Btw we have another mapping for errors that leaves in `src/components/strings/errors.js`. Might be worth moving the error map in `utils`